### ISSUE TITLE
'Hunters in Raid' not properly stored in localStorage (naming mismatch)

### ISSUE THREAD

**Issue by klingo** [watchyoursixx/HunterSim#54]
The input field on the page has the ID '**huntersraid**' and 'onchange' calls '**fightSettings()**'

The '**fightSettings()**' function correctly reads/parses the value, and then stores it in '**huntersraid**' variable before calling '**storeData();**'.
```js
let huntinraid = document.getElementById("huntersraid").value;
huntersraid = parseInt(huntinraid);
```

The '**storeData()**' function then attempts to save it in localStorage:
```js
localStorage.setItem('huntraid',huntersinraid);
```

Issue: It reads it from the variable '**huntersinraid**', but the actual value was stored in '**huntersraid**' - thus it does not get updated in localStorage.

**Comment by watchyoursixx** [comment]
it's kinda confusing I will admit, but it's a different error than you think. Good catch either way though!

- The actual variable name was originally "Huntersinraid", declared in Simulation.js
- The id of the html element is "huntersraid".
- the stored data is called "huntraid".
- the value read in fightSettings() actually uses a undeclared variable "huntersraid"

The issue was when fightSettings() ran it was storing it in a variable that was never declared so it creates it, and does nothing with it. 😅

**Comment by watchyoursixx** [comment]
Fixed in #56
